auto-increment.md

```diff
@@ -101,53 +101,137 @@ In the example above, perform the following operations in order:
 
-### Increment
+### Monotonic
 
-TiDB guarantees that implicitly assigned values of the `AUTO_INCREMENT` column are incremental only in the cluster with a single TiDB instance. That is, for the same auto-increment column, the value assigned earlier is smaller than the value assigned later. However, in a cluster with multiple instances, TiDB cannot guarantee that the auto-increment column is incremental.
+TiDB guarantees that `AUTO_INCREMENT` values are monotonic (always increasing) on a per-server basis. Consider the following example where consecutive `AUTO_INCREMENT` values of 1-3 are generated:
 
-In the example above, if you first execute an `INSERT` statement on instance `B`, and then execute an `INSERT` statement on instance `A`. Because of the behavior of caching auto-increment ID, the auto-increment column might be implicitly assigned `30002` and `2` respectively. The assigned values are not incremental in the time order.
+{{< copyable "sql" >}}
 
-### Monotonic
+```sql
+CREATE TABLE t (a int primary key AUTO_INCREMENT, b timestamp NOT NULL DEFAULT NOW());
+INSERT INTO t (a) VALUES (NULL), (NULL), (NULL);
+SELECT * FROM t;
+```
+
+```sql
+Query OK, 0 rows affected (0.11 sec)
+
+Query OK, 3 rows affected (0.02 sec)
+Records: 3  Duplicates: 0  Warnings: 0
+
++---+---------------------+
+| a | b                   |
++---+---------------------+
+| 1 | 2020-09-09 20:38:22 |
+| 2 | 2020-09-09 20:38:22 |
+| 3 | 2020-09-09 20:38:22 |
++---+---------------------+
+3 rows in set (0.00 sec)
+```
+
+The `AUTO_INCREMENT` sequence might appear to _jump_ dramatically if an insert is performed against a different TiDB server. This is caused by each server having their own cache of `AUTO_INCREMENT` values:
 
-In a cluster with multiple TiDB instances, the `AUTO_INCREMENT` assigned values are **only** monotonic on a per-server basis.
+{{< copyable "sql" >}}
 
-Take the following table as an example:
+```sql
+INSERT INTO t (a) VALUES (NULL);
+SELECT * FROM t;
+```
 
 ```sql
-create table t (a int primary key AUTO_INCREMENT)
+Query OK, 1 row affected (0.03 sec)
+
++---------+---------------------+
+| a       | b                   |
++---------+---------------------+
+|       1 | 2020-09-09 20:38:22 |
+|       2 | 2020-09-09 20:38:22 |
+|       3 | 2020-09-09 20:38:22 |
+| 2000001 | 2020-09-09 20:43:43 |
++---------+---------------------+
+4 rows in set (0.00 sec)
 ```
 
-Execute the following statement on the table:
+A new insert against the initial TiDB server generates the `AUTO_INCREMENT` value of `4`. This is because the initial TiDB server still has space left in the `AUTO_INCREMENT` cache. The sequence of values can not be considered globally monotonic, because the value of `4` was inserted after the value of `2000001`:
```

Comment:
```suggestion
A new `INSERT` operation against the initial TiDB server generates the `AUTO_INCREMENT` value of `4`. This is because the initial TiDB server still has space left in the `AUTO_INCREMENT` cache for allocation. In this case, the sequence of values cannot be considered globally monotonic, because the value of `4` is inserted after the value of `2000001`:
```